about-blockcore.md

```diff
@@ -0,0 +1,26 @@
+**What is Blockcore?**
+
+- Blockcore is a unified repository for all developers of C# full node blockchain technology.
```

Comment:
Ok this is really good, some proposed changes.

- Blockcore is a dotnet core platform to build decentralized networks based on the Bitcoin protocol 
- Blockcore is a platform to build Layer 1 consensus networks based on the Bitcoin protocol,  built on the dotnet core framework and written entirely in C#. blockcore aim to maintain a and keep up to date and alternative Bitcoin implementation.